### PR TITLE
Johha/request page

### PR DESCRIPTION
**Pull request by johanHardon**
made requestpage and corrected the css. It is also refactored.

### CODE REVIEW

File: carRental/src/pages/Request/CarGroupTable.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+const CarGroupTable = ({ numberOfCarGroups }) => {
```
Comment (Evgenios95) [inline carRental/src/pages/Request/CarGroupTable.jsx:1]: Add a comment on top of this component, that it's dummy and that's why we didn't choose to refactor it.

**Review by Evgenios95 — CHANGES_REQUESTED** [review]

File: carRental/src/utils/parse-functions/cloudFunctions.js [hunk 2]
```diff
@@ -20,3 +20,18 @@ export const getBookingsWithState = async (formData, setNumberOfBookings) => {
...
+
+export const getCarGroupsBooked = async (formData, setNumberOfCarGroups) => {
+  const office = await getOfficePointer(formData.rentalOffice);
+  const bookingState = await getBookingStatePointerForEdit("waiting");
+  const params = {
+    rentalOffice: office,
+    bookingState: bookingState,
+  };
+  try {
+    const fetchCargroups = await Parse.Cloud.run("carGroupBooked", params);
```
[the remark this answers is not in the document]
Comment (johanHardon) [reply carRental/src/utils/parse-functions/cloudFunctions.js]: done

File: carRental/src/utils/parse-functions/cloudFunctions.js [hunk 3]
```diff
@@ -20,3 +20,18 @@ export const getBookingsWithState = async (formData, setNumberOfBookings) => {
     console.log(error);
   }
 };
+
+export const getCarGroupsBooked = async (formData, setNumberOfCarGroups) => {
```
[the remark this answers is not in the document]
Comment (johanHardon) [reply carRental/src/utils/parse-functions/cloudFunctions.js]: done

File: carRental/src/pages/Request/private/RequestTable.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
+const RequestTable = () => {
```
[the remark this answers is not in the document]
Comment (johanHardon) [reply carRental/src/pages/Request/private/RequestTable.jsx:3]: done

File: carRental/src/pages/Request/CarGroupTable.jsx (hunk 1 above)
Comment (johanHardon) [reply carRental/src/pages/Request/CarGroupTable.jsx:3]: done

**Review by Evgenios95 — APPROVED** [review]

**Pull request merged**